### PR TITLE
Wip/create registration function

### PR DESCRIPTION
**Pull request by JayOfferdahl**
Created user login logout as well as registration workflows.

### CODE REVIEW

File: YourToDo/templates/auth/loggedin.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class="container">
+        <h1>Hey {{ user.username }}, you logged in!<h1>
```
Comment (bendavidsonku) [inline YourToDo/templates/auth/loggedin.html]: We should change this message to be something else in my opinion. Maybe "Hey {username}, you're logged in!" or not have a loggedin.html and redirect to the homepage

File: YourToDo/templates/auth/login.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class="container">
+        {% if form.errors %}
+            <p>Sorry, that's not a valid username or password</p>
+        {% endif %}
+
+        <form action="/auth/" method="post">{% csrf_token %}
+            <label for="username">User name:</label>
```
Comment (bendavidsonku) [inline YourToDo/templates/auth/login.html]: "User name" to "Username" (one word)

File: YourToDo/templates/auth/registration_success.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class="container">
+        <h1>Thanks, You have registered!</h1>
```
Comment (bendavidsonku) [inline YourToDo/templates/auth/registration_success.html]: you*

**Comment by bendavidsonku** [comment]
I'm going to pull this down and update all the comments I made, @JayOfferdahl, so we can merge this into master quickly and continue development. Let me know if this is okay with you.

**Comment by JayOfferdahl** [comment]
Go for it, that sounds good.

**Pull request merged**